packages/ui-inputs/src/buttons.stories.tsx

```diff
@@ -192,22 +235,25 @@ export const Workshop: Story = () => {
             <Button label={text("Label", "Label")}
                     isActive={boolean("Active", false)}
                     disabled={boolean("Disabled", false)}
-                    type={select(label, options, defaultValue) as 'default' | 'cta' | 'borderless'}
+                    type={select(typeLabel, typeOptions, typeDefaultValue) as 'default' | 'cta' | 'borderless'}
+                    layout={select(layoutLabel, layoutOptions, 'horizontal') as 'horizontal' | 'vertical'}
                     onClick={action('clicked')}></Button>
         </div>
         <div className="col">
             <Button disabled={boolean("Disabled", false)}
                     icon={<Plus24Icon/>}
                     label={text("Label", "Label")}
-                    type={select(label, options, defaultValue) as 'default' | 'cta' | 'borderless'}
+                    type={select(typeLabel, typeOptions, typeDefaultValue) as 'default' | 'cta' | 'borderless'}
+                    layout={select(layoutLabel, layoutOptions, 'horizontal') as 'horizontal' | 'vertical'}
                     isActive={boolean("Active", false)}
                     onClick={action('clicked')}></Button>
         </div>
         <div className="col">
             <Button isProcessing={true}
```

Comment:
please adjust the button to replace show the other processing icon for cta in disabled mode

![image](https://user-images.githubusercontent.com/4751797/70651336-498f0c00-1c59-11ea-8ac6-68b76bc2f7c8.png)

Reply:
fixed